src/ophys_etl/transforms/sine_dewarp.py

```diff
@@ -0,0 +1,606 @@
+import numpy as np
+import os
+import h5py
+import math
+import time
+import json
+import argparse
+import logging
+import multiprocessing
+import functools
+
+
+# Default parameters
+minvalue = 0.0
+maxlevel = 65535  # 16bit
```

Comment:
should be a defaulted parameter or locally-scoped variable in `xdewarp`. I don't see a reason for it to be global.